### ISSUE TITLE
Got error with default LFS_SCHEME(https)

### ISSUE THREAD

**Issue by PeterDaveHello** [github/lfs-test-server#16]
Both browser and git lfs client got this error, and by the way, even I didn't set LFS_SCHEME to http,
I can visit the panel via http protocol, is this a bug?
```
Error for POST https://localhost:8080/temp/test/objects

Post https://localhost:8080/temp/test/objects: tls: oversized record received with length 20527
goroutine 16 [running]:
github.com/github/git-lfs/lfs.Stack(0x0, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/lfs/errors.go:85 +0x89
github.com/github/git-lfs/commands.logPanic(0x7f938c1eac30, 0xc2080aa840, 0x7f938c049a00, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:148 +0xf17
github.com/github/git-lfs/commands.handlePanic(0x7f938c1eac30, 0xc2080aa840, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:100 +0x6c
github.com/github/git-lfs/commands.LoggedError(0x7f938c1eac30, 0xc2080aa840, 0xc208034040, 0x37, 0x0, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:65 +0x89
github.com/github/git-lfs/commands.Panic(0x7f938c1eac30, 0xc2080aa840, 0xc208034040, 0x37, 0x0, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:75 +0x68
github.com/github/git-lfs/commands.pushCommand(0x983f80, 0xc208022450, 0x2, 0x3)
        /Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/command_push.go:120 +0x572
github.com/spf13/cobra.(*Command).execute(0x983f80, 0xc208034080, 0x3, 0x4, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/spf13/cobra/command.go:355 +0x17c
github.com/spf13/cobra.(*Command).findAndExecute(0x984680, 0xc20800e010, 0x4, 0x4, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/spf13/cobra/command.go:334 +0xd3
github.com/spf13/cobra.(*Command).Execute(0x984680, 0x0, 0x0)
        /Users/rick/github/git-lfs/.vendor/src/github.com/spf13/cobra/command.go:408 +0x7b0
github.com/github/git-lfs/commands.Run()
        /Users/rick/github/git-lfs/.vendor/src/github.com/github/git-lfs/commands/commands.go:80 +0x2a
main.main()
        /Users/rick/github/git-lfs/cmd/git-lfs.go:8 +0x1a
```

**Comment by rubyist** [comment]
Yeah, that's kind of a bug. I think we'll remove the `LFS_SCHEME` portion and just use http. At one point some of this code was deployed with a proper ssl setup and I added `LFS_SCHEME` to aid local development, but now this is just a test server. Not using ssl will reenforce the notion that it shouldn't be used in production :smile:

Thanks for the report, I'll push out an update :soon:!

**Comment by PeterDaveHello** [comment]
:+1:

**Comment by rubyist** [comment]
Fixed in #17 and a new release pushed. Thanks!

**Comment by PeterDaveHello** [comment]
Thanks!
